### DISCUSSION TITLE
Running API without homebridge

### DISCUSSION THREAD

**Discussion by neviskevin** [dgreif/ring#665] [Q&A] [answered]
Is there a way to run the API without homebridge? I am trying to run this on an AWS EC2 instance eventually to connect to my house, so I just want to be able to see when my contact sensor triggers on my cli, I have the ring doorbell and security system.

**Comment by dgreif** [comment, marked as answer]
You can definitely use the API without homebridge 😄.  See https://github.com/dgreif/ring#setup-and-config for some basic setup, and https://github.com/dgreif/ring/blob/master/examples/example.ts for a basic example of interacting with the API.  I don't have any examples specifically for watching contact sensors, but everything should be fairly discoverable if you use TypeScript.
